src/plugins/controls/public/control_group/control_group_renderer/control_group_renderer.tsx

```diff
@@ -171,10 +173,12 @@ export const ControlGroupRenderer = ({
         getRuntimeStateForChild: () => {
           return runtimeState$.getValue();
         },
+        compressed,
       })}
       onApiAvailable={(controlGroupApi) => {
         const controlGroupRendererApi: ControlGroupRendererApi = {
           ...controlGroupApi,
+          compressed: compressed ?? true,
```

Comment:
This line is not needed. `compressed` should not be something exposed from the ControlGroupApi. Its only used internally by the control group. Its value comes from the parent. Consumers do not need to know about this implementation detail.